.gitignore

```diff
@@ -0,0 +1,3 @@
+build/
+slurm*.out
+Testing/*.xml
```

Comment:
Happy to have changes to .gitignore for build/, although I'd probably remove slurm.out and Testing/